vendor/unicornstudio/unicornStudio.umd.js

```
@@ -2464,27 +2464,34 @@
         u = a || r || n || l || o;
       return ((e.local.isDynamicCache = u), u);
     }
-    const a = [
+    const r = [
         "trackMouse",
         "axisTilt",
         "trackMouseMove",
         "rotationTracking",
         "lightTracking",
       ].some((t) => e.hasOwnProperty(t) && 0 !== e[t]),
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)